test/test-utils.js

```diff
@@ -0,0 +1,35 @@
+import chai from 'chai';
+import nock from 'nock';
+import { baseUrl } from '../src/utils';
+
+///////////////////////////////////////////////////////////////////////////////
+// Short-hand for comparing two results when testing:
+
+const equal = (a, b) => {
+  return chai.expect(a).to.deep.equal(b);
```

Comment:
Ik zie dit chainen in geen enkele test gebruikt worden, terwij de Chai deep equality check bijna elke test juist gebruikt wordt.Prioriteit heeft dat de tests zo leesbaar mogelijk moeten blijven.